tests/management/group/test_view.py

```diff
@@ -311,6 +326,46 @@ def test_add_group_roles_system_policy_create_success(self):
         self.assertEqual(roles[0].get('description'), self.role.description)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    def test_system_flag_update_on_add(self):
+        """Test that adding a role to a platform_default group flips the system flag."""
+        url = reverse('group-roles', kwargs={'uuid': self.defGroup.uuid})
+        client = APIClient()
+        test_data = {'roles': [self.roleB.uuid, self.dummy_role_id]}
+
+        response = client.post(url, test_data, format='json', **self.headers)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertFalse(Group.objects.filter(uuid=self.defGroup.uuid).first().system)
```

Comment:
Maybe a nit, but if you wanted to just reference `self.defGroup.system` directly here again, you could also do something like:

```
self.defGroup.refresh_from_db()
self.assertFalse(self.defGroup.system)
```

Reply:
I did not know that! Thanks! I'll use that instead.